doc/nrf/ug_contribution.rst

```diff
@@ -0,0 +1,142 @@
+.. _ug_contribution:
+
+Contributing to the |NCS|
+*************************
+
+The |NCS| is maintained by Nordic Semiconductor, but it is open to community contributions.
+
+As a general practice, we encourage to make small, controlled changes.
+This practice simplifies reviewing, merging, and rebasing.
+In addition, it keeps the change history clear and clean.
+
+When contributing to the |NCS|, you should provide as much information as you can about your change.
+Before submitting, you must also update the appropriate documentation and test your changes thoroughly.
+
+The general GitHub workflow that is used by |NCS| developers is based on a combination of Git commands on the command line and browser interaction with GitHub.
+As usual with Git, there are multiple ways of getting a task done.
+The following steps show a typical workflow:
+
+1. :ref:`dm-wf-fork`.
```

Comment:
I added a link to the documentation we already have for this ( https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/ug_dev_model.html#fork-a-repository-of-the-ncs ). Not sure if we want to keep the substeps - they are a bit simpler than the full instructions.
I changed "upstream" to "ncs" though - I think it's a really bad idea to tell people to call their NCS upstream "upstream" in one set of instructions and "ncs" in another ...

Reply:
I agree, good idea 👍